misc/release.py

```diff
@@ -243,9 +230,9 @@ def main():
     if args.host_only:
         hostos, hostarch = hostOsArch()
         logger.info('will only run %s %s', hostos, hostarch)
-    with open('.version') as fin:
-        version = fin.read().strip()
-    ldflags = compile_version_opts(version, allow_mismatch=args.fake_release)
+    version = args.version
+    relversion = args.relversion
+    ldflags = compile_version_opts(version)
     for goos, goarch, debarch in osArchArch:
```

Comment:
This looks like cross-compiling stuff, we should make sure we don't set GOOS/GOARCH anymore